OmniDB/OmniDB_app/views/memory_objects.py

```diff
@@ -268,14 +271,28 @@ def get_database_tab_object(
         # Instead of waiting for garbage collector to clear existing connection,
         # put it in the list of to be removed connections and let the cleaning
         # thread close it
-        if (p_tab_object['omnidatabase']):
+        if p_tab_object['omnidatabase']:
             to_be_removed.append(p_tab_object['omnidatabase'])
 
-
         p_tab_object['omnidatabase'] = v_database_new
 
     # Try to open connection if not opened yet
     if p_attempt_to_open_connection and (not p_tab_object['omnidatabase'].v_connection.v_con or p_tab_object['omnidatabase'].v_connection.GetConStatus() == 0):
         p_tab_object['omnidatabase'].v_connection.Open()
 
     return p_tab_object['omnidatabase']
+
+
+def superuser_required(function):
+    def wrap(request, *args, **kwargs):
+        v_session = request.session.get('omnidb_session')
+
+        if v_session.v_super_user:
+            return function(request, *args, **kwargs)
+        else:
+            v_return = {'v_data': 'You must be superuser to manage users.', 'v_error': True}
```

Comment:
Since there is a chance that this decorator may be used for other cases, the message should be more generic.
Smth like "You must be superuser to perform this operation".